shell/platform/darwin/macos/framework/Source/FlutterExternalTextureGL.mm

```diff
@@ -32,7 +32,7 @@ - (instancetype)initWithFlutterTexture:(id<FlutterTexture>)texture {
 }
 
 - (int64_t)textureID {
-  return reinterpret_cast<int64_t>(self);
+  return reinterpret_cast<int64_t>(_texture);
```

Comment:
@stuartmorgan I couldn't come up with a reason to return self rather than the texture pointer. This also helps us test this better.